### PR TITLE
Add pushing of dev helm charts into dev directory

### PR DESCRIPTION
**Pull request by kkujawa-sumo**
###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: ci/build.sh [hunk 1]
```diff
@@ -110,7 +110,7 @@ function push_helm_chart() {
   git fetch origin-repo
   git checkout gh-pages
 
-  helm repo index --url https://sumologic.github.io/sumologic-kubernetes-collection/ --merge ./index.yaml "${sync_dir}"
+  helm repo index --url https://sumologic.github.io/sumologic-kubernetes-collection/ --merge "${chart_dir}"index.yaml "${sync_dir}"
```
Comment (sumo-drosiek) [inline ci/build.sh:113]:
```suggestion
  helm repo index --url https://sumologic.github.io/sumologic-kubernetes-collection/ --merge "${chart_dir}index.yaml" "${sync_dir}"
```

File: ci/build.sh [hunk 2]
```diff
@@ -150,13 +150,13 @@ build_docker_image "${DOCKER_TAG}" || (echo "Error during building docker image"
 
 if [ -n "${DOCKER_PASSWORD}" ] && [ -n "${TRAVIS_TAG}" ]; then
   push_docker_image "${VERSION}"
-  push_helm_chart "${VERSION}"
+  push_helm_chart "${VERSION}" "./"
```
Comment (sumo-drosiek) [inline ci/build.sh:153]:
```suggestion
  push_helm_chart "${VERSION}" "."
```

Comment (kkujawa-sumo) [reply ci/build.sh:153]: It seems that it doesn't work without / but it could be "" instead of "./". What do you think?

Comment (sumo-drosiek) [reply ci/build.sh:153]: Could you add "/" to the line 113? I edited the suggestion but too late as I see 😅

File: ci/build.sh [hunk 3]
```diff
@@ -110,7 +110,7 @@ function push_helm_chart() {
   git fetch origin-repo
   git checkout gh-pages
 
-  helm repo index --url https://sumologic.github.io/sumologic-kubernetes-collection/ --merge ./index.yaml "${sync_dir}"
+  helm repo index --url https://sumologic.github.io/sumologic-kubernetes-collection/ --merge "${chart_dir}index.yaml" "${sync_dir}"
```
Comment (sumo-drosiek) [inline ci/build.sh:113]:
```suggestion
  helm repo index --url https://sumologic.github.io/sumologic-kubernetes-collection/ --merge "${chart_dir}/index.yaml" "${sync_dir}"
```

File: ci/build.sh (hunk 2 above)
Comment (kkujawa-sumo) [reply ci/build.sh:153]: Ok, I will add "/" in line 113.

File: ci/build.sh (hunk 3 above)
Comment (kkujawa-sumo) [reply ci/build.sh]: Corrected :)

**Review by sumo-drosiek — APPROVED** [review]
Thanks 😃

**Pull request merged**